cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaRoller.java

```diff
@@ -392,7 +392,7 @@ private boolean podWaitingBecauseOfAnyReasons(Pod pod, Set<String> reasons) {
         return false;
     }
 
-    private int idOfPod(String podName)  {
+    public static int idOfPod(String podName)  {
```

Comment:
I think that instead of making this public and sharing the KafkaRoller, you should move it to some `*Util` class.

Reply:
Is there any reason why it should be in `operator-common`? It not, `ModelUtils` in `cluster-operator` would be IMHO better.

Reply:
Just moved to `ModelUtils`